Feature by ivan

Ragnazar opened the pull request:
Добавил дто классы; изменил модели, добавил entity; сгенерировал таблицы users и ads

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/Ads.java:
```diff
@@ -0,0 +1,189 @@
+@Table(name = "ads")
+public class Ads {
+
+    @JsonProperty("author")
```
почему у вас на сущностях аннотации json? сущность - это объект БД, он не используется для (де)-сериализации в JSON, для этого у вас DTO

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/Comment.java:
```diff
@@ -1,20 +1,25 @@
+@Table(name = "comments")
 public class Comment {
 
   @JsonProperty("author")
```
почему у вас на сущностях аннотации json? сущность - это объект БД, он не используется для (де)-сериализации в JSON, для этого у вас DTO

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/CreateAds.java:
```diff
@@ -1,4 +1,4 @@
+package ru.skypro.homework.model.entity;
 
 import java.util.Objects;
 import com.fasterxml.jackson.annotation.JsonProperty;
```
почему у вас на сущностях аннотации json? сущность - это объект БД, он не используется для (де)-сериализации в JSON, для этого у вас DTO

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/FullAds.java:
```diff
@@ -1,4 +1,4 @@
+package ru.skypro.homework.model.entity;
 
 import java.util.Objects;
 import com.fasterxml.jackson.annotation.JsonProperty;
```
почему у вас на сущностях аннотации json? сущность - это объект БД, он не используется для (де)-сериализации в JSON, для этого у вас DTO

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/LoginReq.java:
```diff
@@ -1,11 +1,10 @@
+import java.util.Objects;
 
 /**
  * LoginReq
```
а зачем вам сущность такая? она не нужна, этот DTO приходит для авторизации и только

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/NewPassword.java:
```diff
@@ -1,4 +1,4 @@
-package ru.skypro.homework.model;
+package ru.skypro.homework.model.entity;
 
 import java.util.Objects;
```
и эта сущность зачем? она тоже не нужна

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/RegisterReq.java:
```diff
@@ -1,13 +1,12 @@
+import java.util.Objects;
 
 /**
  * RegisterReq
```
эта тоже не нужна сущность

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/ResponseWrapperAds.java:
```diff
@@ -1,4 +1,4 @@
-package ru.skypro.homework.model;
+package ru.skypro.homework.model.entity;
```
это тоже не сущность, а dto

dmitry-bizin commented on src/main/java/ru/skypro/homework/model/entity/ResponseWrapperComment.java:
```diff
@@ -1,4 +1,4 @@
-package ru.skypro.homework.model;
+package ru.skypro.homework.model.entity;
```
это тоже не сущность, а dto

dmitry-bizin commented on src/main/resources/application.properties:
```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/store_db
+spring.datasource.username=admin
+spring.datasource.password=Qwerty123#
+spring.jpa.hibernate.ddl-auto=update
```
хорошо бы попрактиковаться с liquibase вам, тогда здесь стратегию не забудьте поменять на validate

NatShiv committed:
Сделала сущности и сделала таблицы через liquibase
ads (хранит объявления)
authorities (хранит роли пользователей)
comments (хранит комментарии пользователей)
image (хранит картинки объявлений)
users (хранит пароли пользователей)
user (хранит данные пользователей)

Ragnazar committed:
Merge pull request #5 from Ragnazar/feature_by_Dmitry

Feature by dmitry

Ragnazar committed:
Merge branch 'feature_by_Ivan' into feature_by_Natali

Ragnazar committed:
Merge pull request #6 from Ragnazar/feature_by_Natali

Feature by natali

Commit:
поправочка

Look1nDm committed:
Merge pull request #7 from Look1nDm/feature_by_Dmitry

поправочка

NatShiv committed:
Merge branch 'feature_by_Ivan' into feature_by_Dmitry

NatShiv committed:
Merge pull request #8 from Ragnazar/feature_by_Dmitry

Feature by dmitry
